images/base/Dockerfile

```diff
@@ -95,8 +95,12 @@ RUN echo "Ensuring scripts are executable ..." \
       conntrack iptables iproute2 ethtool socat util-linux mount ebtables udev kmod \
       libseccomp2 pigz \
       bash ca-certificates curl rsync \
-      nfs-common \
-    && find /lib/systemd/system/sysinit.target.wants/ -name "systemd-tmpfiles-setup.service" -delete \
+      nfs-common
+
+# copy in static files (configs)
+COPY files/etc/ /etc/
+
+RUN echo "Cleanup temporary files ..." && find /lib/systemd/system/sysinit.target.wants/ -name "systemd-tmpfiles-setup.service" -delete \
```

Comment:
this step cannot be separate, otherwise the deleted files are still in the layer below.

Reply:
Ack, I guess I can change on the next step instead cause we need the configs before enabling kubelet.